src/screens/TripDetailsScreen.tsx

```
@@ -91,16 +143,83 @@
+                onClick={() => handleRemovePayment(payment.id)}
+              >
+                Remove
+              </button>
+            </li>
+          ))}
+        </ul>
+      )}
+
+      {isPaymentModalOpen && (
+        <div className="modal is-active">
+          <div className="modal-background" onClick={() => setIsPaymentModalOpen(false)}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)